helphours/templates/base.html

```diff
@@ -13,6 +13,12 @@
             }
         </style>
 
+        <script>
+            function toggleDropdown() {
+                document.getElementById("dd-content").classList.toggle("toggle-dropdown-content");
+            }
+        </script>
+
```

Comment:
Maybe we could put this inside of the `{% if g.user.is_authenticated %}` block? Just so we don't have to send the script tag over if it's not needed.